azkaban-common/src/test/java/azkaban/DispatchMethodTest.java

```diff
@@ -0,0 +1,23 @@
+package azkaban;
+
+import azkaban.utils.Props;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static azkaban.Constants.ConfigurationKeys.AZKABAN_EXECUTION_DISPATCH_METHOD;
+
+public class DispatchMethodTest {
+    @Test
+    public void testDispatchMethod() {
+        Props props = new Props();
+        props.put(AZKABAN_EXECUTION_DISPATCH_METHOD, "CONTAINERIZED");
+        Assert.assertEquals(DispatchMethod.getDispatchModel(props), DispatchMethod.CONTAINERIZED);
+        props = new Props();
```

Comment:
Nit : Use the same props object and override the value? I know this aint C++, but I am hard wired like that. Feel free to ignore it.

Reply:
Took care of it. Thank you.